pages/getting-started/learning-odata-on-postman.html

```diff
@@ -19,7 +19,42 @@ <h2>How to Learn OData on Postman</h2>
 </li>
 </ul>
 <p>And the resources behind this is on <a href="https://github.com/ODataOrg/Tutorials">OData tutorial Guidelines</a>.</p>
-<p><a href="/assets/postmanCollection.jpg"><img src="/assets/postmanCollection.jpg" alt="Shows the screenshot of postman being used to query an OData endpoint." width="800" height="490" class="aligncenter size-full wp-image-12687" /></a></p>
+<div class="row">
+    <div class="column">
+        <img src="/assets/postmanCollection.jpg" width="800" height="490" onclick="openModal();currentSlide(1)"
+        onenter id="postmanCollection-small" class="hover-shadow cursor" tabindex="0">
+    </div>
+</div>
+  
+<div id="screenshot1" class="modal" >
+    <div class="modal-content" >
+        <div class="screenshot1Lightbox" width="800" height="490">
+            <img src="/assets/postmanCollection.jpg" width="800" height="490" class="postcollection">
+        </div>
+    </div>
+</div>
 <p>As you can see from the picture above, we have created several requests into different categories in the collection with description, HTTP method, request URL, request headers and response payload (if no response, just click the <em>Send</em> button).</p>
 <h2>How to Contribute to the Query Collection</h2>
 <p>Currently, Postman collection doesn't support collaborating on one specific collection URL, so you may not directly contribute to the collection. BUT, you can contribute to the <a href="https://github.com/ODataOrg/Tutorials">GitHub repositories</a> which servers as a central OData tutorials guidelines and we will update the content regularly.</p>
+
+<style>
```

Comment:
@paulodero I'd recommend moving this CSS to _/public/css/site.css_

Reply:
It tried this but the `css` was not taking effect. We could try together.

Reply:
@paulodero you need to update the style tags in head.html to use the local file when debugging.

Reply:
Perhaps we should actually make that change permanent to make it easier for folks

Reply:
@wandeg We can make the permanent fix. Also do it for syntax.css and autocomplete.css

Reply:
Tested and migrated. Works thanks.